debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/MySqlValueConverters.java

```diff
@@ -85,12 +87,32 @@ public SchemaBuilder schemaBuilder(Column column) {
             return Year.builder();
         }
         if (matches(typeName, "ENUM")) {
-            String commaSeparatedOptions = extractEnumAndSetOptions(column, true);
-            return io.debezium.data.Enum.builder(commaSeparatedOptions);
+            List<String> options = extractEnumAndSetOptions(column);
+            StringBuilder commaSeparatedOptions = new StringBuilder();
+            boolean first = true;
+            for (String value:options) {
+                if (first) {
+                    first = false;
+                } else {
+                    commaSeparatedOptions.append(MySqlDdlParser.ENUM_AND_SET_DELIMINATOR);
+                }
+                commaSeparatedOptions.append(value);
+            }
+            return io.debezium.data.Enum.builder(commaSeparatedOptions.toString());
```

Comment:
This could be done more easily with:

```
String commaSeparatedOptions = Strings.join(","options);
```
using the [io.debezium.util.Strings.join(...)](https://github.com/debezium/debezium/blob/master/debezium-core/src/main/java/io/debezium/util/Strings.java#L195) utility.